src/features/virtualized-table-renderers/dateCellRenderer.js

```diff
@@ -0,0 +1,12 @@
+// @flow
+import * as React from 'react';
+import baseCellRenderer from './baseCellRenderer';
+import ReadableTime from '../../components/time/ReadableTime';
+import type { CellRendererParams } from './flowTypes';
+
+const dateCellRenderer = (cellRendererParams: CellRendererParams) =>
```

Comment:
This seems to only support readableTime and don't have too much option to configure to other needs. Can we name this to readableTimeCellRenderer instead?